QML_HUD/vbar.cpp

```
@@ -1,31 +1,92 @@
+#include <QtMath> 
+
+
+VerticalBar::PolygonPoints::PolygonPoints()
+{
+    m_TopLeftCorner = QPoint(-0.5, 0.5);
+    m_TopRightCorner = QPoint(0.5, 0.5);
+    m_BottomRightCorner = QPoint(0.5, -0.5);
+    m_BottomLeftCorner = QPoint(-0.5, -0.5);
+}
+
+VerticalBar::PolygonPoints::PolygonPoints(QPointF&& tl, QPointF&& tr, QPointF&& br, QPointF&& bl)
```

Cppcheck (2.10-2)

[error] There is an unknown macro here somewhere. Configuration is required. If Q_PROPERTY is a macro then please configure it.

(unknownMacro)